### PR TITLE
[Feature] add rtmpose pruned version

### PR DESCRIPTION
**Pull request by LKJacky**
## Motivation

add rtmpose pruned version

### CODE REVIEW

**Comment by Tau-J** [comment]
better to provide a Chinese-version readme

File: projects/rtmpose/README.md [hunk 1]
```diff
@@ -152,9 +152,11 @@ Feel free to join our community group for more help:
...
+|   Config    | Input Size | AP<sup><br>(COCO) | Params(M) | FLOPS(G) | ORT-Latency(ms)<sup><br>(i7-11700) | TRT-FP16-Latency(ms)<sup><br>(GTX 1660Ti) | ncnn-FP16-Latency(ms)<sup><br>(Snapdragon 865) |    Logs    |    Download    |
+| :---------: | :--------: | :---------------: | :-------: | :------: | :--------------------------------: | :---------------------------------------: | :--------------------------------------------: | :--------: | :------------: |
+| RTMPose-s-aic-coco-pruned |  256x192   |       69.4        |   3.43    |   0.35   |                 -                  |                     -                     |                       -                        | [log](./rtmpose/pruning/group_fisher_finetune_rtmpose-s_8xb256-420e_aic-coco-256x192.py) | [model](https://download.openmmlab.com/mmrazor/v1/pruning/group_fisher/rtmpose-s/group_fisher_finetune_rtmpose-s_8xb256-420e_aic-coco-256x192.pth) |
+
+For more details, please refer to [README.md](./rtmpose/pruning/README.md).
```
Comment (Tau-J) [inline projects/rtmpose/README.md:159]: For more details, please refer to [GroupFisher Pruning for RTMPose](./rtmpose/pruning/README.md)

File: projects/rtmpose/README.md [hunk 2]
```diff
@@ -152,9 +152,11 @@ Feel free to join our community group for more help:
...
-| pruning-s-t |  256x192   |       69.4        |   3.42    |   0.34   |                 -                  |                     -                     |                       -                        | Coming soon | Coming soon |
+|   Config    | Input Size | AP<sup><br>(COCO) | Params(M) | FLOPS(G) | ORT-Latency(ms)<sup><br>(i7-11700) | TRT-FP16-Latency(ms)<sup><br>(GTX 1660Ti) | ncnn-FP16-Latency(ms)<sup><br>(Snapdragon 865) |    Logs    |    Download    |
+| :---------: | :--------: | :---------------: | :-------: | :------: | :--------------------------------: | :---------------------------------------: | :--------------------------------------------: | :--------: | :------------: |
+| RTMPose-s-aic-coco-pruned |  256x192   |       69.4        |   3.43    |   0.35   |                 -                  |                     -                     |                       -                        | [log](./rtmpose/pruning/group_fisher_finetune_rtmpose-s_8xb256-420e_aic-coco-256x192.py) | [model](https://download.openmmlab.com/mmrazor/v1/pruning/group_fisher/rtmpose-s/group_fisher_finetune_rtmpose-s_8xb256-420e_aic-coco-256x192.pth) |
```
Comment (Tau-J) [inline projects/rtmpose/README.md:157]: It seems that you share a link of config file to [Logs]

File: projects/rtmpose/rtmpose/pruning/README.md [hunk 3]
```diff
@@ -0,0 +1,117 @@
...
+### Test
+
+```bash
+CUDA_VISIBLE_DEVICES=0,1,2,3,4,5,6,7 PORT=29500 ./tools/dist_test.sh \
+   {config_folder}/group_fisher_{normalization_type}_finetune_{model_name}.py {checkpoint_path} 8
+```
+
+### Deploy
+
+First, we assume you are fimilar to mmdeploy. For a pruned model, you only need to use the pruning deploy config to instead the pretrain config to deploy the pruned version of your model.
```
Comment (Tau-J) [inline projects/rtmpose/rtmpose/pruning/README.md:85]:
```suggestion
For a pruned model, you only need to use the pruning deploy config to instead the pretrain config to deploy the pruned version of your model. If you are not familiar to mmdeploy, it's recommended to refer to [MMDeploy document](https://mmdeploy.readthedocs.io/en/1.x/02-how-to-run/convert_model.html).
```

File: projects/rtmpose/rtmpose/pruning/README_CN.md [hunk 4]
```diff
@@ -0,0 +1,116 @@
...
+
+| Arch                                                                  | Input Size |  AP   | AP<sup>50</sup> | AP<sup>75</sup> |  AR   | AR<sup>50</sup> | Flops | Params |                   ckpt                    |      log       |
+| :-------------------------------------------------------------------- | :--------: | :---: | :-------------: | :-------------: | :---: | :-------------: | :---: | :----: | :---------------------------------------: | :------------: |
+| [rtmpose-s-pruned](./group_fisher_finetune_rtmpose-s_8xb256-420e_coco-256x192.py) |  256x192   | 0.691 |      0.885      |      0.765      | 0.745 |      0.925      | 0.34  |  3.42  | [pruned][rp_sc_p] \| [finetuned][rp_sc_f] | [log][rp_sc_l] |
+| [rtmpose-s-aic-coco-pruned](./group_fisher_finetune_rtmpose-s_8xb256-420e_aic-coco-256x192.py) |  256x192   | 0.694 |      0.884      |      0.771      | 0.747 |      0.922      | 0.35  |  3.43  | [pruned][rp_sa_p] \| [finetuned][rp_sa_f] | [log][rp_sa_l] |
+
+## Get Started
+
+我们需要三个步骤来将 GroupFisher 应用于你的模型，包括剪枝（Prune），微调（Finetune），部署（Deploy）。
+注意：请使用torch>=1.12，因为我们需要fxtracer来自动解析模型。
```
Comment (Tau-J) [inline projects/rtmpose/rtmpose/pruning/README_CN.md:21]:
```suggestion
注意：请使用 torch>=1.12，因为我们需要 fxtracer 来自动解析模型。
```

**Comment by Tau-J** [comment]
LGTM

**Review by ly015 — APPROVED** [review]

**Pull request merged**